lib/opencensus/trace/integrations/faraday_middleware.rb

```diff
@@ -87,12 +87,19 @@ class FaradayMiddleware < Faraday::Middleware
         #     `DEFAULT_SPAN_NAME`
         # @param [#call] sampler The sampler to use when creating spans.
         #     Optional: If omitted, uses the sampler in the current config.
+        # @param [#serialize,#header_name] formatter The formatter to use when
+        #     propagating span context. Optional: If omitted, uses the
+        #     TraceContext formatter.
```

Comment:
If omitted, uses the formatter detected by the span context. If no formatter was detected, defaults to the TraceContext formatter.